osu.Game/Overlays/Settings/Sections/Gameplay/GeneralSettings.cs

```diff
@@ -76,6 +76,11 @@ private void load(OsuConfigManager config)
                 {
                     LabelText = "Score display mode",
                     Bindable = config.GetBindable<ScoringMode>(OsuSetting.ScoreDisplayMode)
+                },
+                new SettingsCheckbox
+                {
+                    LabelText = "Disable Win key during gameplay",
+                    Bindable = config.GetBindable<bool>(OsuSetting.GameplayDisableWinKey)
                 }
```

Comment:
This would appear in mobile platforms as well from the looks of it

Reply:
yep, forgot to add a conditional check to have that setting appear only on Windows.

 But going forward, I guess having a way to register platform-specific settings maybe the right way to dothis instead of polluting the game config manager with platform-specific settings and having platform checks scattered across the code to hide / show these settings depending on the platform, if we were to add more platform-specific stuff in the future.

Reply:
That's another thing to deal with later on, for now keep it simple and just check for it to only appear in windows.